app/plugin_install.go

```diff
@@ -17,12 +17,29 @@ import (
 	"github.com/mattermost/mattermost-server/utils"
 )
 
+const managedPluginFileName = ".filestore"
+const fileStorePluginFolder = "./plugins"
+
 func (a *App) InstallPluginFromData(data model.PluginEventData) {
 	mlog.Info(fmt.Sprintf("InstallPluginFromData. ID: %v, Path: %v", data.Id, data.FileStorePath))
```

Comment:
This can be removed from here, since I only added it for testing purposes.

Reply:
I actually added it and used it for code clarity as per review feedback. It is used within plugin_install.go and plugin.go

Reply:
Should we remove it ? I think it's important to keep in the log that it is installing a plugin due to a cluster event. Maybe demote to debug? @lieut-data thoughts?

Reply:
I was thinking since we're logging it for error cases, this might not be necessary as it'll be evident on the ui if a plugin was installed on a peer.

Reply:
I'm in favour of a log event when a peer installs a cluster on demand -- maybe that already happens in the existing code, though? 0/5

Reply:
🤷‍♂. +1 to demote to `debug` in that case.

Reply:
✅Demoted to `mlog.Debug`